src/OrbitGl/AccessibleTimeGraph.cpp

```diff
@@ -1,50 +0,0 @@
-// Copyright (c) 2020 The Orbit Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
-#include "AccessibleTimeGraph.h"
-
-#include <vector>
-
-#include "AccessibleInterfaceProvider.h"
-#include "AccessibleTrack.h"
-#include "TimeGraph.h"
-#include "Track.h"
-#include "TrackManager.h"
-#include "Viewport.h"
-
-using orbit_accessibility::AccessibilityRect;
-using orbit_accessibility::AccessibilityState;
-using orbit_accessibility::AccessibleInterface;
-
-namespace orbit_gl {
-
-AccessibilityRect TimeGraphAccessibility::AccessibleRect() const {
```

Comment:
I think `TimeGraph` had a larger-then-screen height before, and we wanted to assume it was always filling the complete capture window for E2E test purposes. This should no longer be needed with the refactorings and the new `TrackContainer` - if it still is, we should solve it by either fixing the E2E tests, or implementing `TimeGraph` correctly instead of faking it through accessibility.